Kabalisa/Send-IT-API
# #162311182 protect all the routes

#### What does this PR do?
add JWT authentication to all protected routes.
#### Description of Task to be completed?
* add the checkToken middleware as one of the handlers of protected routes
* restructure the controllers of all protected routes
* test the routes in postman
#### How should this be manually tested?
clone this repository with ```git clone```
run ```npm start```
run the protected routes with and without a token to see the difference.
#### Any background context you want to provide?
This is a correction of a bug because authentication with JWT was supposed to be done in the creation of the routes. this correction is carried out on a bug branch.
#### What are the relevant pivotal tracker stories?
[#162311182](https://www.pivotaltracker.com/story/show/162311182)
